docs/references/errors.md

```diff
@@ -51,57 +51,58 @@ message GreetingRequest {
 }
 ```
 
-## META0003
+## META0003 {#META0003}
 
-Cannot reach the endpoint to execute service discovery. Make sure:
+Cannot reach the deployment to execute service discovery. Make sure:
 
-* The provided `uri` is correct
-* The service endpoint is up and running
-* The Restate runtime can reach the service endpoint through the configured `uri`
+* The provided `uri`/`arn` is correct
+* The deployment is up and running
+* Restate can reach the deployment through the configured `uri`/`arn`
 * If additional authentication is required, make sure it's configured through `additional_headers`
 
-## META0004
+## META0004 {#META0004}
 
-Cannot register the provided service endpoint, because it conflicts with the uri of an already registered service endpoint.
+Cannot register the provided deployment, because it conflicts with the uri of an already registered deployment.
 
-In Restate service endpoints have a unique uri and are immutable, thus it's not possible to discover the same endpoint uri twice. 
-Make sure, when updating a service endpoint, to assign it a new uri. 
+In Restate deployments have a unique uri/arn and are immutable, thus it's not possible to discover the same deployment twice. 
+Make sure, when updating a deployment, to assign it a new uri/arn. 
 
 You can force the override using the `"force": true` field in the discover request, but beware that this can lead in-flight invocations to an unrecoverable error state.  
 
 See the [versioning documentation](https://docs.restate.dev/services/upgrades-removal) for more information.
 
-## META0005
+## META0005 {#META0005}
 
-Cannot propagate endpoint/service metadata to Restate components. If you see this error when starting Restate, this might indicate a corrupted Meta storage.
+Cannot propagate deployment/service metadata to Restate components. If you see this error when starting Restate, this might indicate a corrupted Meta storage.
```

Comment:
Is this one Restate component that tries to propagate that to another Restate component? It's not clear who is doing the 'propagation'.

Reply:
It is a meta error. Therefore, I believe it is the meta that fails to propagate the information to the rest of Restate.